js/pages/agreements.js

```diff
@@ -280,10 +280,10 @@ export const renderAgreements = async () => {
                                 <div class="row">
                                     <div class="col" style="padding-left: 30px; padding-right:30px;">
                                         <div class="row"  style="border:1px solid lightgrey; border-radius:5px;">
-                                            <div class="col-md-2">
+                                            <div class="col-md-2" data-i18n="agreements.surveyIcon">
                                                 <i class="fab fa-wpforms d-none d-md-block" title="Survey Icon" style="margin-left:10px; font-size:50px;color:#c2af7f;"></i>
                                             </div>
-                                            <div class="col-md-10 cnnsentBodyFont2">
+                                            <div class="col-md-10 cnnsentBodyFont2" data-i18n="agreements.stopSharinRecords">
```

Comment:
This should be `Sharing`, right?

Reply:
That is actually the way the translation team tagged it :) but I will change it.